CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -139,6 +168,23 @@ private static int CalculateReadOnlyOffset(BaseCodeItem codeItem)
             return codeItemField.IsReadOnly ? 0 : 1;
         }
 
+        private static string NormalizeName(BaseCodeItem codeItem)
+        {
+            string name = codeItem.Name;
+            var interfaceItem = codeItem as IInterfaceItem;
+            if ((interfaceItem != null) && interfaceItem.IsExplicitInterfaceImplementation)
```

Comment:
This could be re-written as `if (interfaceItem?.IsExplicitInterfaceImplementation ?? false)`
Very debatable which is easier to read, so just food for thought. :)